src/plan/mod.rs

```diff
@@ -37,7 +37,7 @@ pub use plan_constraints::DEFAULT_PLAN_CONSTRAINTS;
 mod tracelocal;
 pub use tracelocal::TraceLocal;
 
-mod transitive_closure;
+pub(crate) mod transitive_closure;
```

Comment:
See comments in `transitive_closure.rs`.  If we move the code to a different module, we don't need to make this change.

Reply:
Done. I have reverted the change.